cmd/remote_enforcer.go

```diff
@@ -0,0 +1,302 @@
+package main
```

Comment:
Why is the directory called cmd and not remote_enforcer ? 

Why does it make sense to have it at the top level and why not as a directory called remote inside enforcer ?